### PR TITLE
[RFR] Improve the service orientation of the project

### PR DESCRIPTION
**Pull request by Luwangel**
- [x] Rename services
- [x] Use the 15 puzzle api image instead of the git submodule
- [x] Setup a posgresql database instead of sqlite

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -10,17 +10,15 @@ services:
...
     image: nginx:alpine
     ports:
       - "80:80"
     volumes:
       - ./app:/app
       - ./docker/nginx/site.conf:/etc/nginx/conf.d/default.conf
-  15-puzzle-go:
-    build: 15-puzzle-go
+  service_puzzle_api:
+    image: luwangel/15-puzzle-api
```
Comment (alexisjanvier) [inline docker-compose.yaml]: :+1:

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: '2'
 
 services:
-  php:
+  service_php:
```
Comment (alexisjanvier) [inline docker-compose.yaml:4]: I'm not a big fan of the `service_` prefix , but okay.

Comment (Luwangel) [reply docker-compose.yaml:4]: What do you prefer ?

Comment (alexisjanvier) [reply docker-compose.yaml:4]: `service_` is ok

**Review by alexisjanvier — APPROVED** [review]

**Pull request merged**